### DISCUSSION TITLE
Adding a deployment guide using podman with quadlet files.

### DISCUSSION THREAD

**Discussion by madscientist16** [authelia/authelia#8096] [Documentation Feedback]
I believe it would be nice to have a deployment guide for podman.

Here's some example quadlet files for running authelia with podman.

Make a pod for running services related authelia for easy networking between containers. For example if I run redis in the same pod I could access it with `localhost:6379` within the authelia container.
### authelia.pod
```
[Pod]
# Authelia
PublishPort=9091:9091
```
### authelia.container
```
[Unit]
Description=Authelia - An open-source authentication server

[Container]
## General
# AutoUpdate=registry allows me to update the container with podman auto-update when the systemd service is running.
AutoUpdate=registry
Image=ghcr.io/authelia/authelia:latest
Pod=auth.pod

## Volumes
# %h is a shortcut to user home directory
Volume=%h/Server/auth-pod/authelia:/config

## Environment Variables
Environment="TZ=Your/Timezone"
# Use podman secrets to store secrets.
Secret=authelia_ldap_password,type=env,target=AUTHELIA_AUTHENTICATION_BACKEND_LDAP_PASSWORD
Secret=authelia_session_secret,type=env,target=AUTHELIA_SESSION_SECRET
Secret=authelia_storage_encryption_key,type=env,target=AUTHELIA_STORAGE_ENCRYPTION_KEY
Secret=smtp_password,type=env,target=AUTHELIA_NOTIFIER_SMTP_PASSWORD
Secret=authelia_jwt_secret,type=env,target=AUTHELIA_IDENTITY_VALIDATION_RESET_PASSWORD_JWT_SECRET
```

**Comment by eriksjolund** [comment]
Just sharing a Podman tip that might be useful when using an HTTP reverse proxy that supports socket activation (e.g. caddy, traefik or nginx). Use [Alternative 1](https://github.com/eriksjolund/podman-networking-docs?tab=readme-ov-file#alternative-1-create-extra-socket-and-use-networkalias) (create extra socket and use `NetworkAlias=`). 

This tip helped a Reddit user when setting up [traefik+authelia+forgejo](https://www.reddit.com/r/selfhosted/comments/1sgrq3n/traefik_authelia_as_oidc_provider_with_forgejo/)  and another Reddit user setting up  [traefik-oidc-auth+pocket-id](https://www.reddit.com/r/podman/comments/1se2xiq/comment/oeocfmw/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)
